app/Models/SocialFeeds.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Boilerplate\Theme\Models;
+
+use JustCoded\WP\Framework\Objects\Model;
+
+class SocialFeeds extends Model {
+
+	public function get_query( $instance = null ) {
+
+		$args = array(
+			'post_type'      => 'socials_feed',
+			'posts_per_page' => - 1,
+		);
+		if ( ! empty( $instance ) ) {
+			foreach ( $instance['social_networks'] as $key => $val ) {
+				if ( ! empty( $val ) ) {
+					$args['socials_category'][] = $key;
+				}
+			}
+		}
```

Comment:
don't think this method should have posts_per_page = -1. Just imagine how many feeds we have... site will just go down...